platforms/extensibility/plugin-development/src/main/java/org/gradle/plugin/devel/GradlePluginDevelopmentExtension.java

```diff
@@ -154,7 +141,6 @@ public SourceSet getPluginSourceSet() {
      *
      * @return the test source sets
      */
-    @ToBeReplacedByLazyProperty
```

Comment:
💅 Can you add `@NotToBeMigratedToLazy` annotation to these with "beacause" message?

Reply:
`@NotToBeMigratedToLazy` is only valid on classes.  Do you mean `NotToBeReplacedByLazyProperty`?

Reply:
Oh, you are right, my bad. I mean `NotToBeReplacedByLazyProperty`. Thanks for adjusting annotations!